frontend/package.json

```diff
@@ -57,7 +57,7 @@
   },
   "scripts": {
     "start": "react-scripts start",
-    "build": "react-scripts build",
+    "build": "CI=false react-scripts build",
```

Comment:
I'm not sure if this change was intended, and if it wasn't then it might be worth a second look to make sure it doesn't break any tests or anything.

Reply:
This was intentional -- It wasn't running the deployments correctly without that